src/components/dialogs/export-dialog.js

```diff
@@ -57,6 +57,7 @@ const ExportDialog = ({
     useEffect(() => {
```

Comment:
The dropdowns of the dialiog are broken due to the changes in commons-ui. 
Some problems with translations

Reply:
seems to be ok now. cant check again please ?